pomdp_py/framework/basics.pxd

```diff
@@ -27,6 +33,12 @@ cdef class State:
 cdef class Observation:
     pass
 
+cdef class Vector(list):
+    pass
+
+cdef class Response:
+    cdef float _reward
+
```

Comment:
Thanks @troiwill. What is the significance of wrapping a float inside `Response`? Why does this change need to happen in `basic.pyx`? This affects all other programs currently using `pomdp-py`. This would not be acceptable.